wgpu-types/src/lib.rs

```diff
@@ -4248,7 +4271,10 @@ pub struct TextureDescriptor<L> {
     pub format: TextureFormat,
     /// Allowed usages of the texture. If used in other ways, the operation will panic.
     pub usage: TextureUsages,
-    // TODO: missing view_formats https://www.w3.org/TR/webgpu/#dom-gputexturedescriptor-viewformats
+    /// Specifies what view format values will be allowed when calling create_view() on this texture.
+    /// Note: Adding a format to this list may have a significant performance impact,
```

Comment:
I'd causion against this verbiage, as afaik srgb-ness doesn't have significant cost.